### ISSUE TITLE
Follow the steps but fail occurs

### ISSUE THREAD

**Issue by jcrecio** [cebor/angular-highcharts#172]
I tried the basic steps to use angular-highcharts:
![highcharts](https://user-images.githubusercontent.com/2377872/39317747-a73ffc80-497c-11e8-8149-aea74cf9dc29.png)

however I´m getting this error:
![error](https://user-images.githubusercontent.com/2377872/39317782-c02fe444-497c-11e8-94be-af2c8ffd838c.JPG)

but in the steps for the basic example there is no provider given
any idea?

thanks in advance!

**Comment by CollinGraf314** [comment]
Did you update your tsconfig.json file to include highcharts types?

tsconfig.app.json : 
```
{
  "extends": "../tsconfig.json",
  "compilerOptions": {
    "outDir": "../out-tsc/app",
    "baseUrl": "./",
    "module": "es2015",
    "types": [
        "highcharts"
    ]
  },
  "exclude": [
    "test.ts",
    "**/*.spec.ts"
  ]
}
```

**Comment by jcrecio** [comment]
Yes, I did. There must be some missing part of documentation for the vanilla start-up I guess.
